src/Icons/doc/index.rst

```diff
@@ -314,49 +298,6 @@ Rendering Icons
        and embeds the downloaded SVG contents in the template #}
     {{ ux_icon('flowbite:user-solid') }}
 
-HTML Syntax
-~~~~~~~~~~~
-
-.. code-block:: html+twig
-
-    <twig:ux:icon name="user-profile" />
-
-    {# Renders "user-profile.svg" #}
-    <twig:ux:icon name="user-profile" class="w-4 h-4" />
-
-    {# Renders "sub-dir/user-profile.svg" (sub-directory) #}
-    <twig:ux:icon name="sub-dir:user-profile" class="w-4 h-4" />
-
-    {# Renders "flowbite:user-solid" from ux.symfony.com #}
-    <twig:ux:icon name="flowbite:user-solid" />
-
-.. note::
-
-    ``symfony/ux-twig-component`` is required to use the HTML syntax.
-
-Default Attributes
```

Comment:
I'd prefer we keep this section.

Reply:
I shortened it to one sentence here: https://github.com/symfony/ux/blob/b954ff73a6a48e2898a1310a6ea30d1e2395837b/src/Icons/doc/index.rst#L92

Together with the explanation in the config section at https://symfony.com/bundles/ux-icons/current/index.html#full-configuration I think it's clear how this works.

Which part would you say is important to keep?

Reply:
I guess I'm not understanding your motivation here. Why are you suggesting removing this section but not Icon Aliases below?

Reply:
> A single sentence would get lost.

Well, yes and no. I you merge everything I'm suggesting, then the page is only half as long ;-)
And if you start expanding the not-so-important parts (like this one), you need to expand/repeat the real important parts too...

And it's still shown *twice*: In the text, and in the config codeblock.

> Why are you suggesting removing this section but not Icon Aliases below?

Mainly cause I haven't looked into it ;-)
And cause I don't understand that feature - so I thought I'd better not touch it...

Reply:
Is the shortest documentation a goal we have? 

Configuring default attributes is a question we had a lot of times, so dedicating some example was decided. It's not only luck or randomness that did drive every choice here 😅